client/src/Components/UserCard/index.tsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import { Card, Image, Rate } from 'antd';
+import { useTranslation } from 'react-i18next';
+import './style.css';
+
+const saleh = {
+  firstName: 'صباح',
+  lastName: 'الرابي',
+  email: 'sabahrabeye@hotmail.com',
+  password: '$2a$10$2DcYGTGfR06hcepi48.RCubqrqMrKyW1yD2RCZN1YU1UeNU/SC3bq',
+  phone: '05928069997292222',
+  whatsapp: '059544545',
+  description: 'اعمل في مهنة الخياطة منذ 20 سنه وقمت  بتفصيل ازياء للمشاهير',
+};
+function UserCard() {
```

Comment:
Build the component on the props will receive, 
So instead of rendering email on services section just build your object to be the same shape of how data will be from your API
And maybe we can add more padding for the whole card, and decrease padding between image and info section